package.json

```diff
@@ -42,10 +43,10 @@
   "license": "MIT",
   "peerDependencies": {
     "@capacitor/core": ">=3.0.0",
-    "@sentry/angular": "7.114.0",
-    "@sentry/angular-ivy": "7.114.0",
-    "@sentry/react": "7.114.0",
-    "@sentry/vue": "7.114.0"
+    "@sentry/angular": "8.9.2",
+    "@sentry/angular-ivy": "8.9.2",
```

Comment:
The v8 of `@sentry/angular-ivy` does not exist, the package is not published anymnore. 

```suggestion
```